setup.cfg

```diff
@@ -0,0 +1,83 @@
+[metadata]
+name = mpl_interactions
+author = Ian Hunt-Isaak
+author_email = ianhuntisaak@gmail.com
+maintainer_email = ianhuntisaak@gmail.com
+description = Matplotlib aware interact functions
+long_description = file: README.md
+long_description_content_type = text/markdown
+project_urls =
+    Tracker = https://github.com/ianhi/mpl-interactions/issues
+    Changelog = https://github.com/ianhi/mpl-interactions/releases
+    Documentation = https://mpl-interactions.rtfd.io
+    Source = https://github.com/ianhi/mpl-interactions
+classifiers =
+    Framework :: Jupyter
+    Framework :: Matplotlib
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+keywords =
+    IPython
+    Jupyter
+    Matplotlib
+    Widgets
+license = BSD
+platforms = Linux, Mac OS X, Windows
+
+[options]
+python_requires = >=3.6, <3.10
+install_requires =
+    matplotlib >= 3.3
+packages = find:
+
+[options.extras_require]
+jupyter =
+    ipywidgets >= 7.5.0, <8
+    ipympl >= 0.5.8
+
+; Developer requirements
+doc =
+    %(jupyter)s
+    jupyter-sphinx
+    mock
+    mpl-playback >= 0.1.1
+    nbsphinx
+    numpydoc
+    pypandoc
+    pytest_check_links
+    recommonmark
+    sphinx >= 1.5
+    sphinx-autobuild
+    sphinx-copybutton
+    sphinx_gallery >= 0.8.2
+    sphinx_rtd_theme
+    xarray
+test =
+    black
+    nbval
+    pandas
+    PyQt5
+    pytest
+    pytest-mpl
+    requests
+    scipy
+    xarray
+dev =
+    %(doc)s
+    %(jupyter)s
+    %(test)s
+    pre-commit
+
+[options.packages.find]
+where = src
```

Comment:
i think this is also leftover?

Reply:
Actually weird that even works

Reply:
hahaha looking at the tests I'm not sure it did